### PR TITLE
task 5 - GET /api/reviews working and tested with error 404 being han…

### PR DESCRIPTION
**Pull request by sarah-janewilson**
…dled

### CODE REVIEW

File: db/models/reviews.model.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const db = require("../../db/connection");
+
+exports.fetchReviews = () => {
+  return db
+    .query(
+      `SELECT owner, title, reviews.review_id, category, review_img_url, reviews.created_at, reviews.votes, designer, COUNT(comments.review_id)::INT as comment_count FROM reviews LEFT JOIN comments ON reviews.review_id = comments.review_id GROUP BY reviews.review_id ORDER BY reviews.created_at DESC;`
```
Comment (dappernerddesigns) [inline db/models/reviews.model.js:6]: Lovely stuff here, well done.

File: db/models/reviews.model.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    )
+    .then((response) => {
+      if (response.rows.length === 0) {
+        return Promise.reject({ status: 404, message: "Path Not Found" });
```
Comment (dappernerddesigns) [inline db/models/reviews.model.js:10]: You already have a custom handler for 404, I don't believe there would be a way to have a 404 here without a bad path  (which you've handled) as this is not requesting by any particular id. If you take it out, does your test for 404 still pass?

**Review by dappernerddesigns — APPROVED** [review]
Nicely handled. I've left some comments to have a think about as without running your code I actually can't be sure. But see if your 404 tests pass without the custom promise reject as at this stage I don't think you need one. Merge and move on when ready.

File: db/models/reviews.model.js (hunk 2 above)
Comment (sarah-janewilson) [reply db/models/reviews.model.js:10]: Yes it does! I will refactor. Thank you Verity!

**Pull request merged**